test/th5_system.c

```diff
@@ -437,6 +437,31 @@ test_h5_basename(void)
     HDfree(path);
 }
 
+static void
+test_h5_strcasestr(void)
+{
+    const char *const haystack = "My test string";
+    char * str = NULL;
```

Comment:
Almost there! Seems clang-format is unhappy,

```
--- ./test/th5_system.c	(original)
+++ ./test/th5_system.c	(reformatted)
@@ -441,7 +441,7 @@
 test_h5_strcasestr(void)
 {
     const char *const haystack = "My test string";
-    char * str = NULL;
+    char             *str      = NULL;
 
     MESSAGE(5, ("Testing H5_strcasestr\n"));
```

Reply:
Let's hope I got it this time. Running clang-format on my machine ended up with different choices where the '*' should go :facepalm: